addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3527,22 +3529,18 @@ msgstr ""
 
 #empty string with id 824
 
-#. Representation of PVR timer state "disabled"
-#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
-msgctxt "#825"
-msgid "Inactive"
-msgstr ""
+#empty string with id 825
 
-#. Notification text to announce that a timer was activated
+#. Notification text to announce that a timer was enabled
 #: xbmc/pvr/timers/PVRTimerInfoTag.cpp
 msgctxt "#826"
-msgid "Timer activated"
+msgid "Timer enabled"
```

Comment:
We already have 19058 "Timer enabled" => https://github.com/xbmc/xbmc/blob/master/addons/resource.language.en_gb/resources/strings.po#L8677

Please use this one and delete 826 (I introduced this one with the serieas recording PR. It should not be used elsewhere)